examples/kubernetes-example-annotationless-config/readme.md

```diff
@@ -0,0 +1,92 @@
+# Add configmap data to a volume Example 
+
+An example that showcase how to generate a kubernetes resource utilizing Spring Boot-specific metadata. 
```

Comment:
I don't see that you explain how to configure a configmap using annotationless mode ?

Reply:
done